server/models/questions/removeEditor.js

```diff
@@ -13,10 +13,10 @@ function removeEditor({ username, editor, questionId }) {
   return new Promise((resolve, reject) => {
     pool.query(
       `DELETE FROM questions_editors a WHERE
-       EXISTS(SELECT 1 FROM (SELECT * FROM questions_editors) b WHERE editor=? AND question_id=?)
+       EXISTS(SELECT 1 FROM (SELECT * FROM questions_editors) b WHERE editor=? AND question_id=? AND access=?)
        AND NOT EXISTS(SELECT 1 FROM questions WHERE question_id=? AND creator=?)
        AND editor=?`,
-      [username, questionId, questionId, editor, editor],
+      [username, questionId, 'write', questionId, editor, editor],
```

Comment:
#### CHANGE

1. the person **`a`** who is removing editor **`b`** (`write` access) should also have `write` access